frontend/drivers/platform_psp.c

```diff
@@ -342,25 +342,30 @@ static void frontend_psp_exec(const char *path, bool should_load_game)
    RARCH_LOG("Attempt to load executable: [%s].\n", path);
 #if defined(VITA)
    RARCH_LOG("Attempt to load executable: %d [%s].\n", args, argp);
+   int ret;
```

Comment:
This would not be C89-compliant. Make sure to declare variables either at the start of a code block OR function.

Reply:
moved the variable to beginning of function